woocommerce/woocommerce
# Fix create wc extension script

### All Submissions:






### Changes proposed in this Pull Request:

Updates the `create-wc-extension` script within the `woocommerce-admin` plugin to run the `create-extension` from the root package instead.

Closes #36890  .






### How to test the changes in this Pull Request:




1. cd plugins/woocommerce-admin
2. npm run create-wc-extension
3. This should run correctly ( this will generate a plugin folder as a sibling to the `woocommerce-monorepo` folder instead of the `woocommerce-admin` folder.



### Other information:




### FOR PR REVIEWER ONLY:
